src/main/webapp/GetCalculate.java

```diff
@@ -0,0 +1,145 @@
+package com.mishep.web.utils;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.ArrayDeque;
+import java.util.Objects;
+import java.util.StringTokenizer;
+
+public class GetCalculate {
+    public static String getResult(HttpServletRequest req, String tokenStr) {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'getResult(HttpServletRequest, String)' has an NPath complexity of 5461](https://app.codacy.com/app/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4589206)